src/logger.js

```diff
@@ -38,72 +36,32 @@ function getStackTrace () {
 }
 
 class Logger {
-  /**
-     * {string} channel The part/module where the message was logged. Like 'Tanslations'
-     */
   constructor (channel) {
     this.channel = channel
   }
 
-  /**
-     * Log a message on **debug** level
-     * @param {string} message the message (human readable)
-     * @param {string} errorCode (optional) machine readable error code (string in snake_case)
-     * @param {string} payload (optional) JSON payload
```

Comment:
@Simon-Laux likes the method docs for his ide, I'm fine with keeping them.

Reply:
But there are no longer any parameters to document in the logger.

Reply:
Maybe you could be a bit more specific on which method that needs this and i can add it back. I personally dislike this jsdoc style and we don't have it anywhere else in the application. Especially when it means writing 5 lines of documentation for a one line method.